src/cmd/utils.go

```diff
@@ -25,6 +25,7 @@ import (
 	"fmt"
 	"io"
 	"os"
+	"os/user"
```

Comment:
I bet this line is causing:
```ShellSession
cmd/utils.go:30:2: user redeclared in this block
cmd/utils.go:28:2: other declaration of user
```

The code was already importing `os/user` a few lines below, so no need to import it again.  :)

Reply:
Now you removed both imports, and it's causing:
```ShellSession
cmd/utils.go:333:38: undefined: user
```